main.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+const defaultTimeout = 5 * time.Second
```

Comment:
`default` makes it sound like it's overridable somehow, but as far as I can see, it isn't?

Reply:
Ah I guess this is to differentiate it against the bootstrapped timeout.